.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,19 @@
+name: Feature request
+description: File a feature request.
+labels: ["feature"]
```

Comment:
I would use the default labels provided by github, so that we cannot miss tagging.

```suggestion
labels: ["enhancement"]
```